src/pybind/mgr/prometheus/module.py

```diff
@@ -432,15 +439,54 @@ def _setup_static_metrics(self):
                 'Number of {} objects'.format(state),
             )
 
+        for check in HEALTH_CHECKS:
+            path = 'healthcheck_{}'.format(check['name'].lower())
+            metrics[path] = Metric(
+                'gauge',
+                path,
+                check['description'],
+            )
+
         return metrics
 
     @profile_method()
     def get_health(self):
+
+        def _get_value(message, delim=" ", word_pos=0):
+            """Extract value from message (default is 1st field)"""
```

Comment:
nit, use single quotes to be consistent with the rest of this source file.

Reply:
ACK